auth/auth/driver/driver.py

```diff
@@ -315,6 +316,7 @@ async def create(self, name):
         self.name = name
 
     async def _delete(self, name):
+        assert name not in ('default', DEFAULT_NAMESPACE)
```

Comment:
On line 550 in this file, we need to change this line to this:

```
if namespace_name is not None and namespace_name != DEFAULT_NAMESPACE
```

This way we can delete a developer from a namespace.